### PR TITLE
NoUnneededFinalMethodFixer - Mark as risky

### PR DESCRIPTION
**Pull request by SpacePossum**
closes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4771

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1159,9 +1159,14 @@ Choose from the list of available rules:
 
 * **no_unneeded_final_method** [@Symfony, @PhpCsFixer]
 
-  A ``final`` class must not have ``final`` methods and ``private`` method must
+  A ``final`` class must not have ``final`` methods and ``private`` methods must
```
Comment (julienfalque) [inline README.rst]: As this should be merged on master, fixing the typo can be extracted from this PR to merge it on 2.15.

**Comment by SpacePossum** [comment]
is there any technical or BC concern for not sending this to 2.15 though?

**Comment by julienfalque** [comment]
No, this is a SemVer requirement: backward compatible new features should be added in minor releases, not patch ones.

**Comment by SpacePossum** [comment]
todo: make rule risky

**Comment by julienfalque** [comment]
👍 but I still believe adding a new option should only happen on master though.

**Comment by SpacePossum** [comment]
done

**Comment by SpacePossum** [comment]
PHPStan in analyzing the fixtures;

```
The command "php -d auto_prepend_file=dev-tools/vendor/autoload.php ./dev-tools/tools/phpstan analyse" exited with 0.
$ if [ -n "$CHANGED_PHP_FILES" ]; then ./dev-tools/vendor/bin/phpmd `echo "$CHANGED_PHP_FILES" | xargs | sed 's/ /,/g'` text phpmd.xml || travis_terminate 1; fi
/home/travis/build/FriendsOfPHP/PHP-CS-Fixer/tests/Fixtures/Integration/set/@Symfony_whitespaces.test-out.php:110	The class finalClass is not named in CamelCase.
```

this is not what we want I hope?

**Pull request merged**